build.sbt

```diff
@@ -1,5 +1,5 @@
-organization := "com.typesafe.sbt"
-sonatypeProfileName := "com.typesafe"
+organization := "com.github.sbt"
+sonatypeProfileName := "com.github.sbt"
```

Comment:
(if the sonatype profile name is equal to the organization I don't think it needs to be specified explicitly - but can't hurt)